semgrep/semgrep/autofix.py

```diff
@@ -45,8 +95,18 @@ def apply_fixes(rule_matches_by_rule: Dict[Rule, List[RuleMatch]]) -> None:
     for _, rule_matches in rule_matches_by_rule.items():
         for rule_match in rule_matches:
             fix = rule_match.fix
-            if fix:
-                filepath = rule_match.path
+            filepath = rule_match.path
+            if fix and fix.startswith("s/"):  # Regex-style fix
+                try:
+                    from_str, to_str = _parse_regex_fix(fix)
+                    _regex_replace(rule_match, from_str, to_str)
+                    modified_files.add(filepath)
+                except Exception as e:
+                    raise e
```

Comment:
Double raise

Reply:
Oh yeah, that was for debugging. Good catch